eos-upgrade-eos2-to-eos3

```diff
@@ -180,12 +180,12 @@ if [ -n "$printers" ]; then
 else
     echo "No printers found"
 fi
-systemctl start cups 2> /dev/null
+systemctl start cups
 
 # Remove drivers downloaded from OpenPrinting, if any, along with
 # the symlinks created under /var/lib/eos-config-printer/ppd.
 ecp_ppd_dir="/var/lib/eos-config-printer/ppd"
-if [ -d $ecp_ppd_dir ] && [ -n "$(ls $ecp_ppd_dir 2> /dev/null)" ]; then
+if [ -d $ecp_ppd_dir ] && [ -n "$(ls $ecp_ppd_dir)" ]; then
```

Comment:
I don't think you should remove this one -- it's already in an `if` condition which means it won't kill the script if it fails, and the error expected here is:

```
ls: cannot access '/tmp/xaxaxa': No such file or directory
```

which would be unsightly.